sources/HUBContentOperationWrapper.m

```diff
@@ -56,29 +58,30 @@ - (void)performOperationForViewURI:(NSURL *)viewURI
                      previousError:(nullable NSError *)previousError
 {
     self.isExecuting = YES;
-    
+
+    id<HUBContentOperationContext> const context = ({
```

Comment:
It’s a GNU C thingy (“[statement expression](https://gcc.gnu.org/onlinedocs/gcc/Statement-Exprs.html)”) that basically creates a new scope and then the last value is “returned”. The use-case here is to “trick” Xcode into indenting the statement in a way that’s nice to read. Which is all good since we use gnu11 as our C dialect :)